concent_api/core/transfer_operations.py

```diff
@@ -127,19 +128,16 @@ def create_file_transfer_token(
     return file_transfer_token
 
 
-def request_upload_status(file_transfer_token_from_database: message.concents.FileTransferToken) -> bool:
+def request_upload_status(
+    file_transfer_token_from_database:    message.concents.FileTransferToken,
+    report_computed_task:                 message.ReportComputedTask
+) -> bool:
     slash = '/'
     assert len(file_transfer_token_from_database.files) == 1
     assert not file_transfer_token_from_database.files[0]['path'].startswith(slash)
     assert settings.STORAGE_CLUSTER_ADDRESS.endswith(slash)
 
-    current_time = get_current_utc_timestamp()
-    file_transfer_token = message.concents.FileTransferToken(
-        token_expiration_deadline       = current_time + settings.TOKEN_EXPIRATION_TIME,
-        storage_cluster_address         = settings.STORAGE_CLUSTER_ADDRESS,
-        authorized_client_public_key    = settings.CONCENT_PUBLIC_KEY,
-        operation                       = 'upload',
-    )
+    file_transfer_token = create_file_transfer_token(report_computed_task, settings.CONCENT_PUBLIC_KEY, 'download')
```

Comment:
@kbeker I remember that we have some discussion about this part in pull request when you have used 'upload' here and I remember you have some explanation for it. Do you maybe remember what was it ?

Reply:
@rwrzesien @dybi Sorry, but I don't remember why I needed there excalty 'upload'. If @dybi tested it locally and on cluster and everything is ok then everything is good. 
@dybi have you tested checking file existance and uploading file to cluster storage?

Reply:
Tested on storage and updated all our tests